src/js/profile/mobile/widget/Calendar.js

```diff
@@ -137,15 +165,41 @@
 							setDays = setDays + 1;
 							leftDays = leftDays - 1;
 
-							if (div.innerHTML === defaultToday.toLocaleString()) { // today selected.
+							if (defaultToday > parseInt(div.innerHTML, 10)) {
+								if (fixMonth === todayMonth && self.options.pastSelection) {
+									div.classList.remove("currentEnableDay");
```

Comment:
How about adding the selectors instead of the string.
classes = {
  CURRENT_ENABLE_DAY: "currentEnableDay",
  ....
}

Reply:
done.